platforms/core-execution/workers/src/integTest/groovy/org/gradle/workers/internal/WorkerExecutorIntegrationTest.groovy

```diff
@@ -226,6 +228,7 @@ class WorkerExecutorIntegrationTest extends AbstractWorkerExecutorIntegrationTes
     }
 
     @Issue("https://github.com/gradle/gradle/issues/10411")
+    @Requires(value = IntegTestPreconditions.NotEmbeddedExecutor, reason = "This test requires isolated daemons")
```

Comment:
This is at this point almost documentation, and a second level assurance that we are not running in an embedded executor.
IMO, this is a bit more visible vs. the line below, where it's denoted mid-function that it will use a forked executor.